src/translations/qterminal_de.ts

```diff
@@ -270,17 +270,17 @@
     <message>
         <location filename="../mainwindow.cpp" line="449"/>
         <source>&amp;BlockCursor</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Blockschreibmarke</translation>
```

Comment:
@OliverBurkardt - erm nicht um jeden Preis - was zum Teufel ist eine Blockschreibmarke wenn so ein Cursor in Form eines gefüllten Blocks gemeint ist?

Reply:
Hallo Alf, 

ich habe mir bei meinen Übersetzungen einige Gedanken gemacht:

"Cursor" heist in erster Linie nur "Zeiger". "Cursor" ist mehrdeutig und kann im Zweifelsfall auch "Mauszeiger" oder einen Zeiger anderer Art beschreiben. "Schreibmarke" ist eindeutiger als das Wort "Zeiger", da es die Aufgabe des "cursors" gezielt beschreibt. Deshalb ist "Schreibmarke" ein sehr passender Begriff.
In diesem Fall hat die Schreibmarke die Form eines Blocks. Deshalb "Blockschreibmarke". 

Ich finde, dass Denglisch umgangssprachlich klingt. Z.B. statt "Dateimanager" kann man "Dateiverwaltung" schreiben, wie es auch in anderen Desktopumgebungen der Fall ist. Das finde ich eine konsistente Übersetzung ins Deutsche. Mit "Dateimanager" würde man, wie auch mit "cursor" verstehen was gemeint ist, aber das Bild einer überlegten und konsistenten Übersetzung ins Deutsche entsteht dadurch nicht, was sich auf das Benutzererlebnis auswirkt.
Neben dem allein verständlichen Teil einer Übersetzung gibt es auch den ästhetischen. Worüber sich natürlich streiten lässt. Ich habe Dir jedoch dazu eine sachlich fundierte Begründung gegeben, warum "Schreibmarke" aus fachlicher sicht besser geeignet ist als "cursor".

Für den "internet-browser" gibt es im Deutschen kein gleichbedeutendes Wort. Dies wäre dies ein Argument den Begriff "Browser" im Deutschen zu belassen.